plugins/techdocs/api-report.md

```diff
@@ -456,4 +456,7 @@ export class TechDocsStorageClient implements TechDocsStorageApi {
     logHandler?: (line: string) => void,
   ): Promise<SyncResult>;
 }
+
+// @public
+export function toLowerMaybe(str: string, config: Config): string;
```

Comment:
If you do choose to move this out to `plugin-techdocs-react`, consider updating the interface of the function? The scope seems quite wide right now ("given the config for Backstage, and this string, return more or else the same string, but possibly lowercased", which feels _very_ generic, and not necessarily appropriate for the plugin to be responsible for)

But maybe if instead of taking a string and returning a string, you took in an entity name/ref and returned the same? That could be a better interface.